docs/scripts/capture_screenshot.py

```
@@ -0,0 +1,160 @@
+def _demo_dir() -> Path:
+    """Pick a short-pathed directory for the demo workflow.
+
+    Prefers the platform tempdir; falls back to ``/tmp`` when the platform
+    tempdir would blow the header's truncation budget (e.g. macOS, where
+    ``tempfile.gettempdir()`` is a long ``/var/folders/...`` path while
+    ``/tmp`` resolves to a 29-character ``/private/tmp``).
+
+    Returns:
+        Directory (not yet created) to copy the demo workflow into.
+    """
+    candidates = [Path(tempfile.gettempdir()), Path("/tmp")]
```

Ruff (0.15.15)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)